pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3072,6 +3072,63 @@ func (c *Controller) trimDoneVolumeRequests(vm *virtv1.VirtualMachine) {
 	vm.Status.VolumeRequests = tmpVolRequests
 }
 
+// validHotplugVolumes checks if ONLY hotpluggable volumes have been changed
+func validHotplugVolumes(oldVMSpec *virtv1.VirtualMachineSpec, vm *virtv1.VirtualMachine) bool {
```

Comment:
As I noted in #13222
> Must be careful in implementing this fix to make sure that if a user directly edits a VM (not via virtctl) to include hotpluggable volumes, the condition is still added.

This function does not handle that case

Reply:
Ah good point, this is because we still don't have declarative hotplug.